### PR TITLE
Added many resources of necessary youtube playlist of Bangla language

### PR DESCRIPTION
**Pull request by FahimFBA**
## What does this PR do?
Add resources

### Description 
I added many Bangla courses which are actually free (Youtube Playlist) and very much helpful.
You can find all of them on YouTube. So all of them are completely free.

### Why is this valuable?
This is valuable because all of them are in Bangla language which are great resources for all of us whose mother tongue are Bengali. Most of us feel comfortable to start learning the basic on the Bengali language. I tried to find many trustworthy and really good resources for that.

### How do we know it's really free?
As I have stated earlier, you can find all of them on YouTube. So all of them are completely free.

### For book lists, is it a book? For course lists, is it a course? 
These all are courses.

## Checklist:
-done!

### CODE REVIEW

**Comment by FahimFBA** [comment]
As I am new in this PR related stuff, only did one earlier; I'm struggling to solve the problems. I'll try again tomorrow.

**Comment by FahimFBA** [comment]
I've resolved the conflicts. Would you please give it a look?

File: books/free-programming-books-bl.md [hunk 1]
```diff
@@ -1,26 +1,119 @@
 ### Index
 
+* [Android](#android)
+* [Bootstrap](#bootstrap)
 * [C](#c)
+* [C++](#c-plus-plus)
 * [Competitive Programming](#competitive-programming)
-* [JavaScript](#JavaScript)
+* [C Sharp](#c-sharp)
```
Comment (eshellman) [inline books/free-programming-books-bl.md:8]: Let me see if I can get the linter to check the alphabetization of this list. C Sharp will come between C and C++ - take a look at the english lists.

File: books/free-programming-books-bl.md [hunk 2]
```diff
@@ -1,26 +1,119 @@
...
+
+* [C# and ASP.NET MVC Full Bangla Tutorial BITM](https://www.youtube.com/playlist?list=PL_g-DE60bXDBpjMPUWGbmCLHnQDIIcw-6) - Learn With Nirash
+* [C# bangla tutorial | Basic to advance in depth bangla course](https://www.youtube.com/playlist?list=PLbC4KRSNcMnqQakB2xlZPoaV6uau4wTIt) - Learn Hunter
+* [C# OOP (object oriented programming) BanglaTutorials](https://www.youtube.com/playlist?list=PLqCbg_KAOnCe1RLKP2SVmSHZOCD-fWe3p) - Asp Dot Net Explorer
+
+
+### CSS
+
+* [CSS3 and CSS4 Weird Parts Bangla Tutorial](https://www.youtube.com/playlist?list=PL_XxuZqN0xVD3oeT3ckKBmnc7krm-SZl2) - Stack Learner
+* [CSS Complete Course in Bangla 2021](https://www.youtube.com/playlist?list=PLgH5QX0i9K3qjCBXjTmv7Xeh8MDUUVJDO) - Anisul Islam
```
Comment (eshellman) [inline books/free-programming-books-bl.md:71]: CSS before CSS3

File: books/free-programming-books-bl.md [hunk 3]
```diff
@@ -1,26 +1,119 @@
...
+
+### Java
+
+* [Java Bangla Tutorials | CORE Java | Complete OOP](https://www.youtube.com/playlist?list=PLgH5QX0i9K3oAZUB2QXR-dZac0c9HNyRa) - Anisul Islam
+* [Java Swing Bangla Tutorials](https://www.youtube.com/playlist?list=PLgH5QX0i9K3rAHKr6IteF5kdgN6BorH9l) - Anisul Islam
 
 
 ### JavaScript
 
 * [হাতেকলমে জাভাস্ক্রিপ্ট](https://zonayed.js.org) - Zonayed Ahmed
```
Comment (eshellman) [inline books/free-programming-books-bl.md:99]: the unicode collation is used, so bangla character come after Z

File: books/free-programming-books-bl.md (hunk 1 above)
Comment (FahimFBA) [reply books/free-programming-books-bl.md:6]: I've replaced c# between C and C++. Would you kindly check that out?

File: books/free-programming-books-bl.md (hunk 2 above)
Comment (FahimFBA) [reply books/free-programming-books-bl.md:70]: Screenshot (886)

I did it now.

File: books/free-programming-books-bl.md (hunk 3 above)
Comment (FahimFBA) [reply books/free-programming-books-bl.md]: Screenshot (887)
I've tried to correct that.

**Comment by eshellman** [comment]
our volunteer team has added just bangla to the list for the alphabetizing linter; I hope it's ok to wait a day to make sure everything works smoothly.

**Comment by FahimFBA** [comment]
> our volunteer team has added just bangla to the list for the alphabetizing linter; I hope it's ok to wait a day to make sure everything works smoothly.

Sure! I hope you would be able to solve that issue.

**Comment by eshellman** [comment]
OK, the linter is working on this file now. check the output of the linter to see what alphabetization the linter wants. I think it whats bangla before roman.

**Comment by eshellman** [comment]
OK, back to green check mark! Give us a while to review the added resources.

**Comment by eshellman** [comment]
Thanks, this looks good. One thing to note- we try to avoid single-item categories. So, consider including jquery under javascript, etc. etc. Look at other lists to see how topics are organized. Remember, the point is to make it easier for users to find resources of interest; if every list has a different organization, it can be confusing. Not as important for a smaller list of course.

**Comment by FahimFBA** [comment]
> Thanks, this looks good. One thing to note- we try to avoid single-item categories. So, consider including jquery under javascript, etc. etc. Look at other lists to see how topics are organized. Remember, the point is to make it easier for users to find resources of interest; if every list has a different organization, it can be confusing. Not as important for a smaller list of course.

I can't thank you enough for doing those all. I understand your suggestions well, and I promise to try to follow each of them properly.

**Pull request merged**